.dockerignore

```diff
@@ -0,0 +1,17 @@
+# Ignore log and temporary files
+*.log
+*.tmp
+
+# Ignore common development files and directories
+ansible
+.github
+.gitpod.yml
+.pytest_cache
+.vscode
+
+
+# Ignore build and configuration files
+Dockerfile
+docker-compose.yml
+.venv
+*.env
```

Comment:
Worth noting that there's a subtle difference between `.gitignore` and `.dockerignore`, such that this for example ignores `*.env` files only in the root directory (unlike `.gitignore` which would ignore in sub-directories too) - we may want to adjust this to: `**/*.env` to make sure we capture all (although it's not necessary here)

Blocking any `**/__pycache__` folders might be a separate nice addition here